src/main/java/uk/gov/hmcts/reform/judicialapi/service/impl/FeatureToggleServiceImpl.java

```diff
@@ -30,8 +30,8 @@ public FeatureToggleServiceImpl(LDClient ldClient, @Value("${launchdarkly.sdk.us
         this.userName = userName;
     }
 
-    /**
-     * add controller.method name, flag name  in map to apply ld flag on api like below
+    /** mapServiceToFlag.
```

Comment:
minor: please remove method name from javadoc, it will be assigned automatically

Reply:
Removed method altogether, it is not needed.